.travis.yml

```diff
@@ -46,7 +46,7 @@ matrix:
         # FIXME: the following unit tests are skipped (TESTFLAGS=-short):
         #        FAIL: TestPidsSystemd: utils_test.go:55: exec_test.go:630: unexpected error: container_linux.go:353: starting container process caused: process_linux.go:326: applying cgroup configuration for process caused: mountpoint for devices not found
         #        FAIL: TestRunWithKernelMemorySystemd: exec_test.go:713: runContainer failed with kernel memory limit: container_linux.go:353: starting container process caused: process_linux.go:326: applying cgroup configuration for process caused: mkdir : no such file or directory
-        - sudo ssh default -t 'sudo -i make -C /vagrant localunittest TESTFLAGS=-short'
+        - sudo ssh default -t 'sudo -i make -C /vagrant localunittest TESTFLAGS='
```

Comment:
So, you need to remove the `FIXME` comment above, as well as `TESTFLAGS`.

IOW

Reply:
AkihiroSuda suggest me to remove the `-short` flag. At the beginning, I just remove the short flag from TestPidsSystemd and TestRunWithKernelMemorySystemd test cases.
https://github.com/opencontainers/runc/pull/2501#discussion_r450071675